<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buscar_exp</class>
 <widget class="QWidget" name="buscar_exp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_logo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>740</y>
     <width>391</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../logo/logo_empresa.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="fondo_sombra">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>940</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>10</y>
     <width>1561</width>
     <height>881</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="l_horaAlta_i">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>840</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>H. Alta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_home">
    <property name="geometry">
     <rect>
      <x>1438</x>
      <y>10</y>
      <width>112</width>
      <height>108</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/icono_home.png</normaloff>../template/black/icono_home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_observ">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>700</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Observaciones</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="l_observ_i">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>735</y>
      <width>991</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_arriba_pacientes">
    <property name="geometry">
     <rect>
      <x>483</x>
      <y>202</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_arriba2.png</normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_arriba2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_cipsns_i">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CIPSNS</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_1apellido_i">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1 Apellido</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_fecha_nacimiento_i">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>F. Nacimiento</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_genero_i">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>810</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GÃ©nero</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_altura_i">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Altura</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_pacientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>471</width>
      <height>481</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="l_txtbuscar">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>122</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Paciente a buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_peso_i">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>811</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Peso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_txtbuscar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ayuda">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>112</width>
      <height>108</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/ayuda.png</normaloff>../template/black/ayuda.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_2apellido_i">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>690</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2 Apellido</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_expedientes">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>176</y>
      <width>391</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Estudios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_abajo_pacientes">
    <property name="geometry">
     <rect>
      <x>483</x>
      <y>610</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_abajo2.png</normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_abajo2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_edad_i">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>779</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ayadir">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>5</y>
      <width>112</width>
      <height>108</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/anyadir_exp.png</normaloff>../template/black/anyadir_exp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_fechaAlta_i">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>840</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>F. Alta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_usb">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>5</y>
      <width>112</width>
      <height>108</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/usb.png</normaloff>../template/black/usb.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_dni_i">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>747</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DNI o NIE</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_imagen">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>210</y>
      <width>440</width>
      <height>470</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_nombre_i">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="list_estudios">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>205</y>
      <width>451</width>
      <height>481</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_arriba_estudios">
    <property name="geometry">
     <rect>
      <x>1015</x>
      <y>202</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_arriba2.png</normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_arriba2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_abajo_estudios">
    <property name="geometry">
     <rect>
      <x>1015</x>
      <y>610</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_abajo2.png</normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_abajo2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="hs_imagen">
    <property name="geometry">
     <rect>
      <x>1109</x>
      <y>691</y>
      <width>442</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="index_label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>172</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_usb_des">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>5</y>
      <width>112</width>
      <height>108</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/usb.png</normaloff>../template/black/usb.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="l_fecha_hora">
   <property name="geometry">
    <rect>
     <x>1295</x>
     <y>969</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_usuario_acceso">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>970</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
